.github/workflows/ci.yml

```diff
@@ -46,12 +47,11 @@ jobs:
 
       - name: Prepare
         run: |
-          sudo apt install fd-find
+          sudo apt update
+          sudo apt install fd-find --yes
           test -d build || {
-            mkdir -p build
-            wget https://github.com/neovim/neovim/releases/download/nightly/nvim.appimage
-            chmod +x nvim.appimage
-            mv nvim.appimage ./build/nvim
+            chmod +x ./.github/workflows/scripts/setup-appimage.sh
+            ./.github/workflows/scripts/setup-appimage.sh
```

Comment:
I think you misunderstood me. I meant making the script that lies in the repo executable, by running `chmod +x` for that script in your local repo and then pushing this change to make it permanent for this file.

Reply:
Oh, executable permissions persist when launching a container with a shared volume? Didn't know that, thanks!